tests/fast-integration/utils/index.js

```diff
@@ -1041,8 +1041,11 @@ function genRandom(len) {
   return res;
 }
 
-function getEnvOrThrow(key) {
+function getEnvOrThrow(key, defValue = "") {
   if (!process.env[key]) {
+    if (defValue != "") {
+      return defValue
+    }
     throw new Error(`Env ${key} not present`);
   }
```

Comment:
I would not do this change. In this case, it does not throw if the env var is not set. Instead, it returns the def value, but only if it is set, which makes the API hard to understand. I would add another function instead, `getEnvOrDefault` that never throws and returns either the env var or default.